io.github.quantum_mutnauq.fast_reader_gtk.yml

```diff
@@ -0,0 +1,38 @@
+app-id: io.github.quantum_mutnauq.fast_reader_gtk
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: fast-reader
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --device=dri
+  - --socket=wayland
+  
+modules:
+  - name: libconfig
+    buildsystem: autotools
+    post-install:
+      - rm -r -f ${FLATPAK_DEST}/share/info/
+    sources:
+      - type: git
+        url: https://github.com/hyperrealm/libconfig.git
+        tag: v1.5
+      - type: shell
+        commands:
+          - autoreconf -i
+
+  - name: FastReader
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DFLATPAK_BUILD=ON
+      - -DFLATPAK_ID=io.github.quantum_mutnauq.fast_reader_gtk
+    sources:
+      - type: git
+        url: https://github.com/Quantum-mutnauQ/Fast-Reader-GTK.git
+        tag: v6.1
+    post-install:
+      - |
+         rm ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/fastreader.svg
+         rm ${FLATPAK_DEST}/share/metainfo/metainfo.xml
```

Comment:
```suggestion
```

maybe actually don't install them with these incorrect name. Given how your build system wrongly special case flatpak maybe this can be fixed too.